### PR TITLE
ref(sentry10): Remove sentry9 group details code

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.jsx [hunk 1]
```diff
@@ -37,21 +36,14 @@ class GroupTags extends React.Component {
     this.fetchData();
   }
 
-  componentDidUpdate(prevProps) {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.jsx]: Why remove this?

File: src/sentry/static/sentry/app/routes.jsx [hunk 2]
```diff
@@ -1346,33 +1338,6 @@ function routes() {
           </Route>
 
           <Redirect from="group/:groupId/" to="issues/:groupId/" />
-          <Route
-            path="issues/:groupId/"
```
Comment (billyvg) [inline src/sentry/static/sentry/app/routes.jsx]: Similar to releases, we should redirect to new URLs

File: src/sentry/static/sentry/app/routes.jsx [hunk 3]
```diff
@@ -975,69 +967,69 @@ function routes() {
...
-                import(/* webpackChunkName: "OrganizationGroupEventDetails" */ './views/groupDetails/organization/groupEventDetails')
+                import(/* webpackChunkName: "OrganizationGroupEventDetails" */ './views/organizationGroupDetails/groupEventDetails')
               }
               component={errorHandler(LazyLoad)}
             />
             <Route
               path="/organizations/:orgId/issues/:groupId/activity/"
               componentPromise={() =>
-                import(/* webpackChunkName: "GroupActivity" */ './views/groupDetails/shared/groupActivity')
+                import(/* webpackChunkName: "GroupActivity" */ './views/organizationGroupDetails/groupActivity')
```
Comment (markstory) [inline src/sentry/static/sentry/app/routes.jsx]: Longer term do we want the `organization` prefix? It won't mean much when we don't have 'project' based views.

File: src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationGroupDetails/groupTags.jsx]: This didn't apply in the organization version (query is always hardcoded - https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/groupDetails/organization/groupTags.jsx#L11). It was to maintain compatibility with how environments in the project version worked

**Review by billyvg — APPROVED** [review]

**Pull request merged**